### ISSUE TITLE
trouble installing

### ISSUE THREAD

**Issue by huckl3b3rry87** [atom-community/markdown-preview-plus#305]
After trying to install I got this:
```
febbo@febbo-HP-ZBook-17-G2:~$ apm install markdown-preview-plus
Installing markdown-preview-plus to /home/febbo/.atom/packages ✗

npm WARN deprecated wrench@1.5.9: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
npm WARN deprecated CSSselect@0.4.1: the module is now available as 'css-select'
npm WARN deprecated CSSwhat@0.4.7: the module is now available as 'css-what'
npm ERR! Linux 4.4.0-92-generic
npm ERR! argv "/usr/share/atom/resources/app/apm/bin/node" "/usr/share/atom/resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/home/febbo/.atom/.apm/.apmrc" "--userconfig" "/home/febbo/.atom/.apmrc" "install" "/tmp/d-117723-11187-1vdwpvi.s5k7y9o1or/package.tgz" "--runtime=electron" "--target=1.3.15" "--arch=x64" "--global-style"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! file /home/febbo/.atom/.apm/rimraf/2.6.1/package/package.json
npm ERR! code EJSONPARSE

npm ERR! Failed to parse json
npm ERR! No data, empty input at 1:1
npm ERR! 
npm ERR! ^
npm ERR! File: /home/febbo/.atom/.apm/rimraf/2.6.1/package/package.json
npm ERR! Failed to parse package.json data.
npm ERR! package.json must be actual JSON, not just JavaScript.
npm ERR! 
npm ERR! This is not a bug in npm.
npm ERR! Tell the package author to fix their package.json file. JSON.parse

npm ERR! Please include the following file with any support request:
npm ERR!     /tmp/apm-install-dir-117723-11187-1kbdvon.yih5rhehfr/npm-debug.log
```
I would have attached the log error file, but it was not where it said it would be.
Thanks!

**Comment by lierdakil** [comment]
I would assume network connectivity issue, or npmjs being uncooperative. If successive attempts fail with the same error message, I would suspect corrupted Atom settings directory (`$HOME/.atom`) and/or corrupted Atom installation.
TL;DR: try again, maybe it'll work? If not, try removing `/home/febbo/.atom/.apm/rimraf`. If it doesn't help, rename `/home/febbo/.atom` and try again? (**WARNING:** removing `~/.atom` directory will **remove all Atom packages and settings**, so making a backup is **strongly** suggested -- you have been warned)

**Comment by lierdakil** [comment]
ETIMEDOUT, closing.
